### DISCUSSION TITLE
Links with MetricsReloaded

### DISCUSSION THREAD

**Discussion by thibaultdvx** [Project-MONAI/MONAI#8082] [General]
Hi,

I am currently particularly interested in your metric module. I know that you are quite close to the **MetricsReloaded** project and I would like to know if eventually all the metrics developed in this project will be directly added to MONAI, or if it will stay as it is, i.e. accessible through wrappers like `MetricsReloadedBinary`?

Thanks,

**Comment by KumoLiu** [comment]
Hi @thibaultdvx, hopefully we will do more integration, but for now we only have some usage example, you can refer to this tutorial: https://github.com/Project-MONAI/tutorials/tree/main/modules/metrics_reloaded.

Hope it helps, thanks.

**Reply by thibaultdvx** [reply]
It does, thanks!
